android/app/build.gradle

```diff
@@ -92,7 +92,7 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion '27.0.1'
```

Comment:
Plugin 2.3.0 needs build tools 25.0.0 or higher, so we'll still need to upgrade this one.

I checked the release notes for that:
https://developer.android.com/studio/releases/build-tools

and it looks quite compatible -- basically bug fixes and support for new language and Android features -- so that's fine.